atef/widgets/config/run_active.py

```diff
@@ -98,3 +101,80 @@ def __init__(self, *args, data: PreparedDescriptionStep, **kwargs):
     def _setup_ui(self) -> None:
         self.title_label.setText(self.bridge.origin.get().name)
         self.desc_label.setText(self.bridge.origin.get().description)
+
+
+class SetValueRunWidget(DesignerDisplay, DataWidget):
+    """
+    Widget for viewing set value step.  Displays Results alongside their
+    respective actions or checks
+    """
+    filename = 'set_value_run_widget.ui'
+
+    actions_table: QtWidgets.QTableWidget
+    checks_table: QtWidgets.QTableWidget
+
+    def __init__(self, *args, data: PreparedSetValueStep, **kwargs):
+        super().__init__(*args, data=data, **kwargs)
+        self._setup_ui()
+
+    def _setup_ui(self) -> None:
+        for table, field, cls in [
+            (self.actions_table, self.data.prepared_actions, ActionRowRunWidget),
+            (self.checks_table, self.data.prepared_criteria, CheckRowRunWidget)
+        ]:
+            for item in field:
+                ins_ind = table.rowCount()
+                action_row = cls(data=item)
+                table.insertRow(ins_ind)
+                table.setRowHeight(ins_ind, action_row.sizeHint().height())
+                table.setCellWidget(ins_ind, 0, action_row)
```

Comment:
I think the buttons here are good candidates for simple tooltips e.g. "Click here to confirm value" on the value confirmation button

Reply:
Addressed in [93849d0](https://github.com/pcdshub/atef/pull/159/commits/93849d0eee9f6b3d14c1a77d6be658f0e66cce93)